src/parsers/common.jl

```diff
@@ -19,28 +19,28 @@ merge!(
     ),
 )
 
-const STRING2PRIMEMOVER = Dict((string(x) => x) for x in instances(PrimeMovers))
+const STRING2PRIMEMOVER =
```

Comment:
The keys used to be all-caps...I guess we should have future-proofed ourselves in some way. Thanks. Do we need to do the same thing to `STRING2FUEL` now?

Reply:
Probably yes

Reply:
Probably should, though those happen to all be uppercase so far

Reply:
Done